windows/secureboot_enable_disable/secureboot_enable_disable.yml

```diff
@@ -31,6 +31,15 @@
             skip_reason: "Not Applicable"
           when: vm_firmware is undefined or vm_firmware | lower != 'efi'
 
+        - name: "Get VBS status of VM"
```

Comment:
Can add a when condiditon "guest_os_ansible_architecture == "64-bit"" for this block.

Reply:
Ok, thanks.